pkg/util/strategicpatch/patch.go

```diff
@@ -31,7 +31,352 @@ import (
 // lists should be merged or replaced.
 //
 // For more information, see the PATCH section of docs/api-conventions.md.
+
+// CreateStrategicMergePatch creates a patch that can be passed to StrategicMergePatch.
```

Comment:
Slight refinement. The definition of a conflict depends on the value of a force parameter to the reconcile method. If force is true, then a conflict occurs if there are any changes or deletions to keys in live are not also changed or deleted in the modified document. If force is false, then a conflict occurs if there are any changes or deletions to keys in live that are not changed or deleted in the same way in the modified document. The force parameter therefore lets the caller choose between failing for any conflicts, and failing only for conflicts that cannot be resolved by letting the user's changes override changes made by automated processes.